test-utils/src/helpers.rs

```diff
@@ -1,7 +1,16 @@
 use crate::types::{Body, HttpResponse, Id, Uri};
+use hyper::service::{make_service_fn, service_fn};
+use hyper::{Request, Response, Server};
 use serde_json::Value;
+use std::convert::Infallible;
 use std::net::SocketAddr;
 
+pub const PARSE_ERROR: &str = "Parse error";
+pub const INTERNAL_ERROR: &str = "Internal error";
+pub const INVALID_PARAMS: &str = "Invalid params";
+pub const INVALID_REQUEST: &str = "Invalid request";
```

Comment:
Note, the specification states: `Invalid Request`  but we use `Invalid request`.

https://github.com/paritytech/jsonrpc/ does the same let's keep like that for now.